Filters/fcl/prolog.fcl

```diff
@@ -73,7 +85,11 @@ DigiCompression : {
 	caloShowerStepTags : @local::DigiCompressionTags.primaryCaloShowerStepTags
 	caloShowerSimTag : @local::DigiCompressionTags.commonCaloShowerSimTag
 	caloShowerStepROTag : @local::DigiCompressionTags.commonCaloShowerStepROTag
-	keptProducts : [ "keep *_compressDigiMCs_*_*" ]
+	keptProducts : [ "keep *_compressDigiMCs_*_*"                      ,
+			 "keep    mu2e::EventWeight_*_*_*"                 ,
```

Comment:
This will defeat compression of GenParticles.

Reply:
I can update this to be `GenParticles_generate_photon_*`, but the `GammaConversionGun` produces a GenParticle (which Andy asked to be in a list for future compression) that has the original photon energy.
https://github.com/michaelmackenzie/Offline/blob/Mu2eIIRMC/EventGenerator/src/GammaConversionGun_module.cc#L368-#L370
(Also, how do I add the line references here?)

This is needed, as the e+e- pair often has a lower energy than the original photon due to energy losses between generation and eventual conversion (e.g. Compton scatter and then a conversion).
This photon energy is used for RMC weights, rather than the e+e- pair energy as with internal conversion generation.

Perhaps in a future PR this can be added to the compression list, removing the need for adding this line.

Reply:
I've updated this to specify the module and tag, so the saved Gen Particle vector won't be saved by this.